build.gradle

```diff
@@ -16,7 +16,7 @@ buildscript {
 }
 
 plugins {
-  //id "com.gradle.build-scan" version "1.9"
+  id "com.gradle.build-scan" version "1.9"
```

Comment:
Does it build?

Reply:
Seems to for me. Building from the AR side.

Reply:
Was accidental though, so if it works the other way I'll change it back